back-end/API/tests/test_employee_jobOffer.py

```
@@ -0,0 +1,470 @@
+        "employmentSchedules": [{ "id": es.id, "description": es.description }],
+        "enterprise": {
+            "name": "Google",
+            "email": "google@gmail.com",
+            "phone": "1234567890",
+            "address": "123 rue google",
+            "cityId": 1
+        },
+    }
+
+    response = client.put(f'/jobOffer/2', json=data)
+    assert response.json['isApproved']==None
```

Ruff (0.15.13)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)